bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/scheduler2/CronJob.java

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright (c) 2014-2017 by the respective copyright holders.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.core.scheduler2;
+
+import java.util.Map;
+
+/**
+ * The software utility Cron is a time-based job scheduler in Unix-like computer
+ * operating systems. People who set up and maintain software environments use
+ * cron to schedule jobs (commands or shell scripts) to run periodically at
+ * fixed times, dates, or intervals. It typically automates system maintenance
+ * or administration—though its general-purpose nature makes it useful for
+ * things like connecting to the Internet and downloading email at regular
+ * intervals.[1] The name cron comes from the Greek word for time, χρόνος
+ * chronos.
+ * <p>
+ * The Unix Cron defines a syntax that is used by the Cron service. A user
+ * should register a Cron service with the {@link CronJob#CRON} property. The
+ * value is according to the {link http://en.wikipedia.org/wiki/Cron}.
+ * <p>
+ *
+ * <pre>
+ * * * * * * * *
+ * | │ │ │ │ │ |
```

Comment:
How does the docblock relate to anything in this interface? Is see no parameters to pass an cronexpression?